packages/components/src/components/button/_button.scss

```diff
@@ -215,14 +215,14 @@
 
   .#{$prefix}--btn--ghost.#{$prefix}--btn--icon-only
     .#{$prefix}--btn__icon
-    path,
+    path:first-of-type,
```

Comment:
instead of relying on the order of the paths should we just target the inner paths?

Reply:
Do you mean `data-icon-path="inner-path"`? That would target the wrong path in the filled versions and it doesn't seem like the non-filled versions have that attribute. I might be misunderstanding?

Reply:
```css
path[data-icon-path="inner-path"] {
  fill: none;
}
```

Reply:
I meant targeting the path directly but not using the same style rule

Reply:
@emyarod updated, much cleaner 💯